pset-iterators.js

```diff
@@ -183,7 +214,12 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 7:')
 
 // Add your code below this line
-
+const shortTermPresidents = presidents.filter(function(president){
+  return president.took_office
+  return president.left_office
+})
+console.log(shortTermPresidents)
+console.log(left_office -= took_office)
```

Comment:
@ChicoMangusta Good attempt here at this solution to use this filter method to return the presidents who served for less than 4 years. You needed to do a little more inside your filter method. The solution below will help give you some further insight.

Take a look at this alternate solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L201)